members/TempReg

```diff
@@ -0,0 +1,73 @@
+bioguide: TempReg
+contact_form:
+  method: post
+  action:
+  steps:
+    - visit: "https://www.regulations.gov/comment?D=NOAA-NOS-2017-0066-0001"
+    - wait:
+      - value: 3
+    - find:
+      - selector: div.GIY1LSJAQC form
+    - wait:
```

Comment:
If a form is being slow, you generally only need a wait step before a find step, not after. The find step already has a built in 5 second delay.